NewForce-at-Mountwest/bangazon-workforce-gnocci
# IT should be able to view, create, and delete computers

Description
On the Home page users can click the Computer Item to see a list of All Computers. Users my also click the create new button and will see a form to create a new Computer.
Users can click the Details link to be directed to view 'Id' 'PurchaseDate' 'Make' 'Manufacturer' and the User can click 'Back to List' Link to go back to the computer home page.
User can click on the 'Delete' link to be redirected to a new page where the user can delete a computer. 

Related Ticket(s)
IT should be able to view, create, and delete computers
https://trello.com/c/KdnUBk4i/8-it-should-be-able-to-view-create-and-delete-computers

Steps to Test
in gitbash cd to banazon-workforce-gnocci
start BangazonWorkforce.sln
press F5 to run browser
in browser click Computer on the navBar
click create new
create new Computer (Make sure you add in a year when doing the date)
